EpiAware/src/latent-processes.jl

```diff
@@ -1,32 +1,25 @@
-const STANDARD_RW_PRIORS =
-    (var_RW_dist = truncated(Normal(0.0, 0.05), 0.0, Inf), init_rw_value_dist = Normal())
-
-
-"""
-    random_walk(n, ϵ_t = missing; latent_process_priors = (var_RW_dist = truncated(Normal(0., 0.05), 0., Inf),), ::Type{T} = Float64) where {T <: Real}
-
-Constructs a random walk model.
-
-# Arguments
-- `n`: The number of time steps.
-- `ϵ_t`: The random noise vector. Defaults to `missing`, in which case it is sampled from the standard multivariate normal distribution.
-- `latent_process_priors`: The prior distribution for the latent process parameters. Defaults to `(var_RW_dist = truncated(Normal(0., 0.05), 0., Inf),)`.
+function default_rw_priors()
```

Comment:
So this was a suggest for what I would expect to see in R. Does it make sense in Julia or was the original more normal?

Reply:
> Also this implementation suggests it will be used interatively (in the scan function). Are we clear that this is more efficient than the usual approach to implementation (exp(log_init_incidence + cumsum(rt)))?

The function is iteratively applied but was _also_ supposed to do the transformation back from the unconstrained domain (default apply `exp`). It wasn't doing that so I've committed a fix.

There won't be an efficiency gain for vectorising this call in Julia (it compiles down to same thing).

The main utility of `scan` here is that all the models come down to the same set of operations.

Reply:
If we were only doing `rt` I'd just use the pattern above.

Reply:
Another option is to apply the (default) `exp` transform inside `random_walk` and abstract away `scan` or not using `scan` inside the `EpiModel` callable definition.

Reply:
Or a mixture of those strategies

Reply:
This way is fine/better. Goal should be zero `const` objects if possible.